docs/guides/admin/docs/modules/paella.player7/url.parameter.md

```diff
@@ -0,0 +1,56 @@
+Paella Player 7 - URL Parameters
+==============================
+
+Parameter      | Example    | Description
+---------------|------------|------------
+**id**         | `SOME-ID`  | Video Id to play
+**time**       | `10m20s`   | Seeks initially automatically to a specified time
+**trimming**   | `1m2s:10m` | Apply a soft trimming to the video
```

Comment:
The **log** parameter and the description is missing.

Reply:
Done!